src/spatialdata_plot/pl/render.py

```diff
@@ -156,27 +158,27 @@ def _render_shapes(
 
     # Determine which method to use for rendering
     method = render_params.method
+
     if method is None:
         method = "datashader" if len(shapes) > 10000 else "matplotlib"
-    elif method not in ["matplotlib", "datashader"]:
-        raise ValueError("Method must be either 'matplotlib' or 'datashader'.")
+
     if method != "matplotlib":
         # we only notify the user when we switched away from matplotlib
-        logger.info(f"Using '{method}' as plotting backend.")
+        logger.info(
+            f"Using '{method}' backend with '{render_params.ds_reduction}' as reduction"
+            " method to speed up plotting. Depending on the reduction method, the value"
+            " range of the plot might change. Set method to 'matplotlib' do disable"
+            " this behaviour."
+        )
 
     if method == "datashader":
         trans = mtransforms.Affine2D(matrix=affine_trans) + ax.transData
```

Comment:
It seems like we could use `trans` defined on line 155 here. Though this line is not changed in this PR perhaps we can change it here.